README.md

```diff
@@ -207,6 +207,26 @@ timestamp.timezone=Europe/Berlin
 timestamp.source=wallclock
 ```
 
+## Development
+
+### Integration testing
+
+Integration tests are implemented using JUnit, Gradle and Docker.
+
+To run them, you need:
+- Docker installed.
+
+Integration testing doesn't require a valid AWS credentials.
+
+To simulate AWS S3 behaviour tests uses [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.
```

Comment:
```suggestion
To simulate AWS S3 behaviour the tests use [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.
```

Reply:
How about without `the`, but with comma?
- [ ] fix grammar